content/en/docs/event-sources/_index.md

```diff
@@ -3,8 +3,16 @@ title: Event Sources
 weight: 5
 ---
 
-Falco can consume events from different sources, and apply rules to these events to detect abnormal behavior. Currently Falco supports the following event sources:
+Falco can consume events from a variety of different sources and apply rules to these events to detect abnormal behavior.
+
+Currently Falco supports the following built-in event sources:
```

Comment:
Currently, Falco supports the following built-in event sources:

Reply:
Sure, fixed.